.github/workflows/CI.yml

```diff
@@ -16,162 +16,61 @@ defaults:
-    # built afterwards (in other jobs that depend on this one).
-    runs-on: ubuntu-20.04
-    outputs: # declare what this job outputs (so it can be re-used for other jobs)
-      # build config
-      # metadata
-      GIT_SHA: ${{ steps.extract_branch.outputs.GIT_SHA }}
-      BRANCH: ${{ steps.extract_branch.outputs.BRANCH }}
-      BRANCH_SLUG: ${{ steps.extract_branch.outputs.BRANCH_SLUG }}
-      VERSION: ${{ steps.get_version.outputs.VERSION }}
-      DATE: ${{ steps.get_datetime.outputs.DATE }}
-      TIME: ${{ steps.get_datetime.outputs.TIME }}
-      DATETIME: ${{ steps.get_datetime.outputs.DATETIME }}
-
-    steps:
-      - name: Check out code
-        uses: actions/checkout@v2.3.4
-        with:
-          fetch-depth: 0 # need to checkout "all commits" for certain features to work (e.g., get all changed files)
-
-      - name: Load CI Environment from .ci_env
-        id: load_ci_env
-        uses: c-py/action-dotenv-to-setenv@v3
-        with:
-          env-file: .ci_env
-
-      - name: Extract branch name
-        id: extract_branch
-        # see https://github.com/keptn/gh-action-extract-branch-name for details
-        uses: keptn/gh-action-extract-branch-name@main
-
-      - name: 'Get Previous tag'
-        id: get_previous_tag
-        uses: "WyriHaximus/github-action-get-previous-tag@v1.0.1"
-      - name: 'Get next patch version'
-        id: get_next_semver_tag
-        uses: "WyriHaximus/github-action-next-semvers@v1.1"
-        with:
-          version: ${{ steps.get_previous_tag.outputs.tag }}
-      - name: Get the version
-        id: get_version
-        env:
-          BRANCH: ${{ steps.extract_branch.outputs.BRANCH }}
-          BRANCH_SLUG: ${{ steps.extract_branch.outputs.BRANCH_SLUG }}
-        shell: bash
-        run: |
-          # determine version
-          GIT_LAST_TAG=${{ steps.get_previous_tag.outputs.tag }}
-          GIT_NEXT_TAG=${{ steps.get_next_semver_tag.outputs.patch }}
-          echo "GIT_LAST_TAG=${GIT_LAST_TAG}, GIT_NEXT_TAG=${GIT_NEXT_TAG}"
-
-          if [[ "$BRANCH" == "release-"* ]]; then
-            # Release Branch: extract version from branch name
-            VERSION=${BRANCH#"release-"}
-            else
-            if [[ "$BRANCH" == "master" ]]; then
-              # master branch = latest
-              VERSION="${GIT_NEXT_TAG}-dev"
-            else
-              # Feature/Development Branch - use last tag with branch slug
-              VERSION="${GIT_NEXT_TAG}-dev-${BRANCH_SLUG}"
-            fi
-          fi
-
-          echo "VERSION=${VERSION}"
-
-          echo "##[set-output name=VERSION;]$(echo ${VERSION})"
-      - name: Get current date and time
-        id: get_datetime
-        run: |
-          echo "::set-output name=DATE::$(date +'%Y%m%d')"
-          echo "::set-output name=TIME::$(date +'%H%M')"
-          echo "::set-output name=DATETIME::$(date +'%Y%m%d')$(date +'%H%M')"
+    uses: keptn/gh-automation/.github/workflows/prepare-ci.yml@feat/prepare_ci
```

Comment:
This should of course be update to use a tag from the keptn/gh-automation repo as soon as the PR over there is merged (keptn/gh-automation#10)

Reply:
done